bin/p2-bootstrap/bootstrap.go

```diff
@@ -72,15 +72,12 @@ func InstallConsul(consulPod *pods.Pod, consulManifest *pods.PodManifest) error
 }
 
 func RegisterBaseAgentInConsul(agentManifest *pods.PodManifest) error {
-	store, err := intent.LookupStore(intent.Options{})
-	if err != nil {
-		return err
-	}
+	store := p2consul.NewStore(p2consul.Options{})
 	hostname, err := os.Hostname()
 	if err != nil {
 		return err
 	}
-	_, err = store.SetPod(hostname, *agentManifest)
+	_, err = store.SetPod(p2consul.Slash(p2consul.INTENT_TREE, hostname, agentManifest.ID()), *agentManifest)
```

Comment:
We should wrap this join in a method specific to the different stores we have instead of forcing all callers to do the `Join` themselves.